diff --git a/setup.py b/setup.py
--- a/setup.py
+++ b/setup.py
@@ -15,13 +15,13 @@
     name="robocasa",
     packages=[package for package in find_packages() if package.startswith("robocasa")],
     install_requires=[
-        "numpy==1.23.3",
+        "numpy==1.23.5",
         "numba==0.56.4",
         "scipy>=1.2.3",
         "mujoco==3.2.6",
         "pygame",
         "Pillow",
-        "opencv-python",
+        "opencv-python-headless",
         "pyyaml",
         "pynput",
         "tqdm",
